NxUI.lua

```diff
@@ -3456,7 +3456,7 @@ function Nx.Button:Init()
 	f:Hide()
 
 	local t = f:CreateTexture()
-	t:SetColorTexture (Nx.Util_str2rgba ("0.06|0.06|0.250|1"))
+	t:SetColorTexture (Nx.Util_str2rgb ("0.06|0.06|0.250"))
```

Comment:
I don't know what the bug here is - apparently `SetColorTexture` still accepts an alpha value, but this is not working anymore. I spent like 2.5 hours debugging this and even tested this code:

```
local r, g, b, a = Nx.Util_str2rgba ("0.06|0.06|0.250|1")
t:SetColorTexture(r, g, b, a)
```

which doesn't work (error like in #223).
I printed out the values of `r,g,b,a` and they were
```
0.06
0.06
0.250
1
```
which is what I expected. Harcoding the values like
```
t:SetColorTexture(0.06, 0.06, 0.250, 1)
```
also works and
```
t:SetColorTexture(r, g, b, 1)
```
also.

It's only that the alpha value does not seem to be working if passed as a variable.
I also tried setting it to `1.0` and `0.5`: Hardcoded works, but not as a variable.